src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -749,7 +745,10 @@ symvmFromCommand cmd = do
           error $ "contract not found."
         Just contract' ->
           return $
-            vm1 cdlen calldata' callvalue' caller' (contract'' & set EVM.storage store)
+            vm1 calldata' callvalue' caller'
+                (contract''
+                   & set EVM.storage store
+                   & set EVM.origStorage store)
```

Comment:
this was a bug in the previous implementation right?

Reply:
yeah, I think so

Reply:
What tests would catch this?

Reply:
Maybe comparing gas usage of `seth --run-tx` and `seth send`